server/channels/app/file.go

```diff
@@ -100,7 +106,14 @@ func (a *App) TestFileStoreConnection() *model.AppError {
 func (a *App) TestFileStoreConnectionWithConfig(cfg *model.FileSettings) *model.AppError {
 	license := a.Srv().License()
 	insecure := a.Config().ServiceSettings.EnableInsecureOutgoingConnections
-	backend, err := filestore.NewFileBackend(filestore.NewFileBackendSettingsFromConfig(cfg, license != nil && *license.Features.Compliance, insecure != nil && *insecure))
+	var backend filestore.FileBackend
+	var err error
+	if license.IsCloud() && a.Config().FeatureFlags.CloudDedicatedExportUI {
+		// TODO - allow configure insecure to false? Probably not, since cloud
+		backend, err = filestore.NewFileBackend(filestore.NewExportFileBackendSettingsFromConfig(cfg, license != nil && license.IsCloud() && *license.Features.Compliance, false))
```

Comment:
What is the value of `EnableInsecureOutgoingConnections` in cloud? Can we just rely on that config value?